docs/architecture/adr-026-ibc-client-recovery-mechanisms.md

```diff
@@ -51,6 +52,9 @@ We elect not to deal with chains which have actually halted, which is necessaril
 
     Previously, `AllowUpdateAfterExpiry` and `AllowUpdateAfterMisbehaviour` were used to signal the recovery options for an expired or frozen client, and governance proposals were not allowed to overwrite the client if these parameters were set to false. However, this has now been deprecated because a code migration can overwrite the client and consensus states regardless of the value of these parameters. If governance would vote to overwrite a client or consensus state, it is likely that governance would also be willing to perform a code migration to do the same.
 
+    In addition, `TrustingPeriod` was initally not allowed to be updated by a client upgrade proposal. However, due to the number of situations experienced in production where the `TrustingPeriod` of a client should be allowed to be updated because of ie: initial misconfiguration for a canonical channel, governance should be allowed to update this client parameter. 
```

Comment:
```suggestion
    In addition, `TrustingPeriod` was initally not allowed to be updated by a client upgrade proposal. However, due to the number of situations experienced in production where the `TrustingPeriod` of a client should be allowed to be updated because of, for example, initial misconfiguration for a canonical channel, governance should be allowed to update this client parameter. 
```

Reply:
i will leave ie: for brevity

Reply:
Although [Google's developer documentation style guide](https://developers.google.com/style/word-list) recommends not to use the abbreviation (_Instead, use phrases like for example or such as. Too many people mix up e.g. and i.e._), then I think it would be better to use `e.g.`, since you are giving an example (instead of clarifying the previous statement), right?